docs/root/modules/github/schema.md

```diff
@@ -152,6 +158,12 @@ WRITE, MAINTAIN, TRIAGE, and READ ([Reference](https://docs.github.com/en/graphq
     (GitHubUser)-[:OUTSIDE_COLLAB_{ACTION}]->(GitHubRepository)
     ```
 
+- GitHubUsers are members of an organization.  In some cases there may be a user who is "unaffiliated" with an org, for example if the user is an enterprise owner, but not member of, the org.
```

Comment:
Add in a note here to highlight the significance of this. This is a user that can do [insert actions here] and yet not show up on member lists of the GitHub org.